### PR TITLE
VTCCA-4056: gds migration start page

### PR DESCRIPTION
**Pull request by zahiriqbal**
gds migration for start page
bug fixed for vtcca 4045 for assessment page

### CODE REVIEW

File: app/views/dashboard/assessmentDetails.scala.html [hunk 1]
```diff
@@ -111,28 +111,28 @@
     </div>
 }
 
+@noRateableValueContent = {
+    <p>@messages("rv.na.p.1")</p>
```
Comment (PeterPerhac) [inline app/views/dashboard/assessmentDetails.scala.html:115]: p tags need govuk-body class

File: app/views/start.scala.html [hunk 2]
```diff
@@ -17,55 +17,51 @@
...
-            <p>@messages("start.registration.p.3")</p>
+        </h1>
 
-            <ul class="list list-bullet">
-                <li>@messages("start.registration.youNeed.1")</li>
-                <li>@messages("start.registration.youNeed.2")</li>
-                <li>@messages("start.registration.youNeed.3")</li>
-            </ul>
+        <p class="govuk-body">@Html(Messages("start.registration.p.1", routes.Login.show))</p>
+        <p>@Html(Messages("start.registration.p.2", routes.Application.addUserToGG))</p>
```
Comment (PeterPerhac) [inline app/views/start.scala.html:41]: more p tags that need the govuk class

File: app/views/start.scala.html [hunk 3]
```diff
@@ -17,55 +17,51 @@
...
-            <br/>
+        @govukDetails(Details(
+        summary = Text(messages("start.registration.IdontHave")),
+        content = HtmlContent(messages("start.registration.IdontHave.answer"))
+        ))
 
-            <legend class="form-label-bold">
-                <span class=" label-span">@Html(Messages("create-account.type.question"))</span>
-            </legend>
+        <h4 class="govuk-label govuk-label--m">@Html(Messages("create-account.type.question"))</h4>
```
Comment (PeterPerhac) [inline app/views/start.scala.html:59]: not sure if it's meant to have double dash... `--m` ?

File: app/views/dashboard/assessmentDetails.scala.html (hunk 1 above)
Comment (darrell-plant) [reply app/views/dashboard/assessmentDetails.scala.html:115]: I think not in this case as this block is used inside a govUkDetails, but will check

File: app/views/start.scala.html [hunk 4]
```diff
@@ -17,55 +17,51 @@
...
 
-            <p>@Html(Messages("start.registration.individual.hyperlink", config.ggRegistrationUrl, config.dashboardUrl("home"), "voa"))</p>
-            <p>@Html(Messages("start.registration.organisation.hyperlink", config.ggRegistrationUrl, "organisation", config.dashboardUrl("home"), "voa"))</p>
+        <p>@Html(Messages("start.registration.individual.hyperlink", config.ggRegistrationUrl, config.dashboardUrl("home"), "voa"))</p>
```
Comment (PeterPerhac) [inline app/views/start.scala.html:61]: again p tag missing css class

File: app/views/start.scala.html (hunk 2 above)
Comment (darrell-plant) [reply app/views/start.scala.html:41]: <p> inherits class `govuk-body` from the enclosing `div` - we shouldn't need to add classes to <p> evry time we use them

File: app/views/start.scala.html (hunk 3 above)
Comment (darrell-plant) [reply app/views/start.scala.html:59]: `--m` is correct

File: app/views/start.scala.html (hunk 2 above)
Comment (darrell-plant) [reply app/views/start.scala.html]: the div wrapper maybe bad practice, so have removed and will add class to <p>'s

**Comment by darrell-plant** [comment]
LGTDz 👍

**Comment by PeterPerhac** [comment]
👍  LGTM

**Pull request merged**